src/platform/ios.rs

```diff
@@ -90,6 +108,18 @@ pub trait WindowBuilderExtIOS {
 
     /// Sets the valid orientations for the `Window`.
     fn with_valid_orientations(self, valid_orientations: ValidOrientations) -> WindowBuilder;
+
+    /// Sets whether the `Window` prefers the home indicator hidden.
+    ///
+    /// The default is to prefer showing the home indicator.
+    fn with_prefers_home_indicator_hidden(self, hidden: bool) -> WindowBuilder;
+
+    /// Sets the screen edges for which the system gestures will take a lower priority than the
+    /// applications touch handling.
```

Comment:
```suggestion
    /// application's touch handling.
```
(there's another instance of this on line 54)

For platform-specific stuff like this, it's also great to include a link to [the relevant docs](https://developer.apple.com/documentation/uikit/uiviewcontroller/2887512-preferredscreenedgesdeferringsys) (the same goes for `with_prefers_home_indicator_hidden`).